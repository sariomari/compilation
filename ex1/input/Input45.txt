,2
(intarray]+mnil,=(int};:=:=>nclass	};308
-"vVlP"262
]class{/*!
?(
]
;X]
X!Z(!?F*///]6(.32
2901
class{-(
><nilclass+:=voidu>classclass>[(,
extends(//n
+>+{:=nil>Carraystringy[
void7
:=<:=(20
)int
int=<;[<M.<//96(/*B
/*/):=nil;int<"kCrhyv"(>18
}extends[.751
685
6678
/*?L!/)]]]?*/UGD,WKiarray=],)>	//]d) =532
2700

[+>}+nil..]nilx1h5T(""-31
.class/*v
}];*/aQc)string
	arrayvoid{class<=string11
/*H;!6*//*
2;}?]*/string-Q6meT/*j!!
v]]?}!gK])]9}(}4;
?r;*/)([nilr628,class=stringnil	{)[9089
>ts>35
[)//
)],;[)extends[]string6
	int=459
nil,58
=/**/class.;""	>extends>	8289
})array 	]})(--,void+intclassstringnil/*]*/<-+3
48
class-}<{)574
:=67
}+uIextends);852
10
;{string=extends-voidiJ;,,/*)Jfo
(/6!;!!*/],}:=}28
//?]n=<stringnil)//F;(4?82ht4;Q82{>/*
D(
8M;!}
((}g*/<(void-array)array{;class,1
,{
"rBNlQ"=<=nilvoid172
extendsint{//i]/;
int{/*!)*/1
,;).]750
void9
,:=//;}}(
]-	24
(890
+extends8827
>	0
..:=+:=6390
nil2
stringstring<(/*;*/+int/*v?!!U?;o?*/ /*3w);?)
;}]
]}d2]?;]8*/{string0
string-[/*
B(!(;xU}?]
i})4
);Qy!*/array(		{array	;,voidnil8
(>.=>extends<[,;nil(}},65
q29tO6K//+:=nil2
{
545
/*]])1!u5;)
*/]+<string,{array243
[extendsvoid},{stringLq1oT"hua")<array9
}</*;8]!(70H(?T()
!}a}

!*/"CI")
:=8120
{5943
}array:=void]}47

8
/*
?*//*B;}p;]N)HU*/258
string6
:=,""
.extends.+="boxo";}""-int738
9
stringextends3
=string]
<nil""h;extends	 "UHi"]27
6427
//
>T1G6Pint}>
voidJ;/*u*/}8919
""string})].:=HlJSEcx)	>)/*s(;;}]9Y?c?!;]?}?F}!*/int,(extends[:=-,//]!(?HY!!!;?{{-]{.extendstkU13
void-
/*!*/;<4
],21
 {>{array8
{	<("s"y2classclass]int[2370
nilstring]}<-nil+46
-//
/*?c];4]]}*/<+[<-<4
;];}stringnil21
].}
void.)QTstring= ]3645
980
;,nil,=7008
537
}737
7302
-<nilint>string)"JDU"[([-[.=string +string{//P[54
>6870
.+(array-+-]:=)-classextends+:="yV"888

]67
	5513
void==.nil//;)6

//]9(m}}[;;}}!wp
NNc)void389
]>9083
472
	
{.//1?2!})]n!;array[extends</*;
])]}}!(I*/int }}]nil:=.:=
)nil0
//!)>-9121
=rSPcintclass;//{//!}
extends=void):=nil9385
.;extends/*n]
]w?!.5?I)!0()*/OB[(<)85
string]int+	.{]7
arrayextendsclass45
.""<>163
>int=stringclass[/*;()](}};uo];
?)t;6!(}(;x!r)(?*/stringint{"pEmeZPX"<)int]classextends+string(7828
-void<//?d+extends<:=int}981
:=;g4B//
-,<<=/*
;}!)*/nilnilextends
 -int]47
//));
}[/*?(5;;?}
!
];?
(;;b
]?!]i*/string940
(array/*0X(4}
}*/""2834
class)620
}//??i;898
+})<}/*5

(J?)
]
m*/+;extends614
"TtK"((nilclass;{string.+.;6
class;-string"U".S389VCwint
37
(/*G!?*/-;extendsvoid-void<extends/*x!};]?;??1F?];(OB(;T*/[[.+array{k} /*!i?*//*{a}!}*/{-/*;}
o)]?
*/string"uu"-4
,735
853
array//class(;stringclass]+//:=)voidclass.)nil--"urq"
}{]""-Tostring=
intarray,5
arraynil:=array,>{{/*}S];*/,=class;/*]}]!)6!*/>;4992
.1
[
//}];(F
extendsvFsHBGMtK]>"Wz"3
[arrayarray,-nilvoid<[	t0
3
<
}7818
 [}}]nilnil(}extendsextends<}class({47
extends}//P!!]
<==5679
+-<[ class80
Mtqj4KkBUV