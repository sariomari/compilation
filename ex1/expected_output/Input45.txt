COMMA[1,1]
INT(2)[1,2]
LPAREN[2,1]
ID(intarray)[2,2]
RBRACK[2,10]
PLUS[2,11]
ID(mnil)[2,12]
COMMA[3,1]
EQ[3,2]
LPAREN[3,3]
TYPE_INT[3,4]
RBRACE[3,7]
SEMICOLON[3,8]
ASSIGN[3,9]
ASSIGN[3,11]
GT[3,13]
ID(nclass)[3,14]
RBRACE[3,21]
SEMICOLON[3,22]
INT(308)[3,23]
MINUS[4,1]
STRING("vVlP")[4,2]
INT(262)[4,8]
RBRACK[5,1]
CLASS[5,2]
LBRACE[5,7]
DOT[10,1]
INT(32)[10,2]
INT(2901)[11,1]
CLASS[12,1]
LBRACE[12,6]
MINUS[12,7]
LPAREN[12,8]
GT[13,1]
LT[13,2]
ID(nilclass)[13,3]
PLUS[13,11]
ASSIGN[13,12]
ID(voidu)[13,14]
GT[13,19]
ID(classclass)[13,20]
GT[13,30]
LBRACK[13,31]
LPAREN[13,32]
COMMA[13,33]
EXTENDS[14,1]
LPAREN[14,8]
PLUS[15,1]
GT[15,2]
PLUS[15,3]
LBRACE[15,4]
ASSIGN[15,5]
NIL[15,7]
GT[15,10]
ID(Carraystringy)[15,11]
LBRACK[15,24]
ID(void7)[16,1]
ASSIGN[17,1]
LT[17,3]
ASSIGN[17,4]
LPAREN[17,6]
INT(20)[17,7]
RPAREN[18,1]
TYPE_INT[18,2]
TYPE_INT[19,1]
EQ[19,4]
LT[19,5]
SEMICOLON[19,6]
LBRACK[19,7]
LT[19,8]
ID(M)[19,9]
DOT[19,10]
LT[19,11]
LPAREN[20,1]
RPAREN[22,4]
ASSIGN[22,5]
NIL[22,7]
SEMICOLON[22,10]
TYPE_INT[22,11]
LT[22,14]
STRING("kCrhyv")[22,15]
LPAREN[22,23]
GT[22,24]
INT(18)[22,25]
RBRACE[23,1]
EXTENDS[23,2]
LBRACK[23,9]
DOT[23,10]
INT(751)[23,11]
INT(685)[24,1]
INT(6678)[25,1]
ID(UGD)[27,10]
COMMA[27,13]
ID(WKiarray)[27,14]
EQ[27,22]
RBRACK[27,23]
COMMA[27,24]
RPAREN[27,25]
GT[27,26]
EQ[28,2]
INT(532)[28,3]
INT(2700)[29,1]
LBRACK[31,1]
PLUS[31,2]
GT[31,3]
RBRACE[31,4]
PLUS[31,5]
NIL[31,6]
DOT[31,9]
DOT[31,10]
RBRACK[31,11]
ID(nilx1h5T)[31,12]
LPAREN[31,20]
STRING("")[31,21]
MINUS[31,23]
INT(31)[31,24]
DOT[32,1]
CLASS[32,2]
ID(aQc)[33,6]
RPAREN[33,9]
TYPE_STRING[33,10]
ID(arrayvoid)[34,2]
LBRACE[34,11]
CLASS[34,12]
LT[34,17]
EQ[34,18]
ID(string11)[34,19]
TYPE_STRING[39,8]
MINUS[39,14]
ID(Q6meT)[39,15]
RPAREN[46,6]
LPAREN[46,7]
LBRACK[46,8]
ID(nilr628)[46,9]
COMMA[46,16]
CLASS[46,17]
EQ[46,22]
ID(stringnil)[46,23]
LBRACE[46,33]
RPAREN[46,34]
LBRACK[46,35]
INT(9089)[46,36]
GT[47,1]
ID(ts)[47,2]
GT[47,4]
INT(35)[47,5]
LBRACK[48,1]
RPAREN[48,2]
RPAREN[49,1]
RBRACK[49,2]
COMMA[49,3]
SEMICOLON[49,4]
LBRACK[49,5]
RPAREN[49,6]
EXTENDS[49,7]
LBRACK[49,14]
RBRACK[49,15]
ID(string6)[49,16]
TYPE_INT[50,2]
EQ[50,5]
INT(459)[51,1]
NIL[53,1]
COMMA[53,4]
INT(58)[53,5]
EQ[54,1]
CLASS[54,6]
DOT[54,11]
SEMICOLON[54,12]
STRING("")[54,13]
GT[54,16]
EXTENDS[54,17]
GT[54,24]
INT(8289)[54,26]
RBRACE[55,1]
RPAREN[55,2]
ARRAY[55,3]
RBRACK[55,10]
RBRACE[55,11]
RPAREN[55,12]
LPAREN[55,13]
MINUS[55,14]
MINUS[55,15]
COMMA[55,16]
TYPE_VOID[55,17]
PLUS[55,21]
ID(intclassstringnil)[55,22]
LT[55,44]
MINUS[55,45]
PLUS[55,46]
INT(3)[55,47]
INT(48)[56,1]
CLASS[57,1]
MINUS[57,6]
RBRACE[57,7]
LT[57,8]
LBRACE[57,9]
RPAREN[57,10]
INT(574)[57,11]
ASSIGN[58,1]
INT(67)[58,3]
RBRACE[59,1]
PLUS[59,2]
ID(uIextends)[59,3]
RPAREN[59,12]
SEMICOLON[59,13]
INT(852)[59,14]
INT(10)[60,1]
SEMICOLON[61,1]
LBRACE[61,2]
TYPE_STRING[61,3]
EQ[61,9]
EXTENDS[61,10]
MINUS[61,17]
ID(voidiJ)[61,18]
SEMICOLON[61,24]
COMMA[61,25]
COMMA[61,26]
RBRACK[64,10]
COMMA[64,11]
RBRACE[64,12]
ASSIGN[64,13]
RBRACE[64,15]
INT(28)[64,16]
EQ[66,1]
LT[66,2]
ID(stringnil)[66,3]
RPAREN[66,12]
GT[67,1]
LT[70,7]
LPAREN[70,8]
TYPE_VOID[70,9]
MINUS[70,13]
ARRAY[70,14]
RPAREN[70,19]
ARRAY[70,20]
LBRACE[70,25]
SEMICOLON[70,26]
CLASS[70,27]
COMMA[70,32]
INT(1)[70,33]
COMMA[72,1]
LBRACE[73,1]
STRING("rBNlQ")[74,1]
EQ[74,8]
LT[74,9]
EQ[74,10]
ID(nilvoid172)[74,11]
ID(extendsint)[75,1]
LBRACE[75,11]
TYPE_INT[76,1]
LBRACE[76,4]
INT(1)[77,4]
COMMA[78,1]
SEMICOLON[78,2]
RPAREN[78,3]
DOT[78,4]
RBRACK[78,5]
INT(750)[78,6]
ID(void9)[79,1]
COMMA[80,1]
ASSIGN[81,1]
RBRACK[82,1]
MINUS[82,2]
INT(24)[82,4]
LPAREN[83,1]
INT(890)[83,2]
PLUS[84,1]
ID(extends8827)[84,2]
GT[85,1]
INT(0)[85,3]
DOT[86,1]
DOT[86,2]
ASSIGN[86,3]
PLUS[86,5]
ASSIGN[86,6]
INT(6390)[86,8]
ID(nil2)[87,1]
ID(stringstring)[88,1]
LT[88,13]
LPAREN[88,14]
PLUS[88,20]
TYPE_INT[88,21]
LBRACE[94,3]
ID(string0)[94,4]
TYPE_STRING[95,1]
MINUS[95,7]
LBRACK[95,8]
ARRAY[104,5]
LPAREN[104,10]
LBRACE[104,13]
ARRAY[104,14]
SEMICOLON[104,20]
COMMA[104,21]
ID(voidnil8)[104,22]
LPAREN[105,1]
GT[105,2]
DOT[105,3]
EQ[105,4]
GT[105,5]
EXTENDS[105,6]
LT[105,13]
LBRACK[105,14]
COMMA[105,15]
SEMICOLON[105,16]
NIL[105,17]
LPAREN[105,20]
RBRACE[105,21]
RBRACE[105,22]
COMMA[105,23]
INT(65)[105,24]
ID(q29tO6K)[106,1]
PLUS[107,1]
ASSIGN[107,2]
ID(nil2)[107,4]
LBRACE[108,1]
INT(545)[109,1]
RBRACK[113,3]
PLUS[113,4]
LT[113,5]
TYPE_STRING[113,6]
COMMA[113,12]
LBRACE[113,13]
ID(array243)[114,1]
LBRACK[115,1]
ID(extendsvoid)[115,2]
RBRACE[115,13]
COMMA[115,14]
LBRACE[115,15]
ID(stringLq1oT)[115,16]
STRING("hua")[115,27]
RPAREN[115,32]
LT[115,33]
ID(array9)[115,34]
RBRACE[116,1]
LT[116,2]
STRING("CI")[120,3]
RPAREN[120,7]
ASSIGN[121,1]
INT(8120)[121,3]
LBRACE[122,1]
INT(5943)[122,2]
RBRACE[123,1]
ARRAY[123,2]
ASSIGN[123,7]
TYPE_VOID[123,9]
RBRACK[123,13]
RBRACE[123,14]
INT(47)[123,15]
INT(8)[125,1]
INT(258)[130,4]
ID(string6)[131,1]
ASSIGN[132,1]
COMMA[132,3]
STRING("")[132,4]
DOT[133,1]
EXTENDS[133,2]
DOT[133,9]
PLUS[133,10]
EQ[133,11]
STRING("boxo")[133,12]
SEMICOLON[133,18]
RBRACE[133,19]
STRING("")[133,20]
MINUS[133,22]
ID(int738)[133,23]
INT(9)[134,1]
ID(stringextends3)[135,1]
EQ[136,1]
TYPE_STRING[137,1]
RBRACK[137,7]
LT[138,1]
NIL[138,2]
STRING("")[138,5]
ID(h)[138,7]
SEMICOLON[138,8]
EXTENDS[138,9]
STRING("UHi")[138,18]
RBRACK[138,23]
INT(27)[138,24]
INT(6427)[139,1]
GT[141,1]
ID(T1G6Pint)[141,2]
RBRACE[141,10]
GT[141,11]
ID(voidJ)[142,1]
SEMICOLON[142,6]
RBRACE[142,12]
INT(8919)[142,13]
STRING("")[143,1]
TYPE_STRING[143,3]
RBRACE[143,9]
RPAREN[143,10]
RBRACK[143,11]
DOT[143,12]
ASSIGN[143,13]
ID(HlJSEcx)[143,15]
RPAREN[143,22]
GT[143,24]
RPAREN[143,25]
TYPE_INT[148,14]
COMMA[148,17]
LPAREN[148,18]
EXTENDS[148,19]
LBRACK[148,26]
ASSIGN[148,27]
MINUS[148,29]
COMMA[148,30]
LBRACE[149,1]
LBRACE[149,2]
MINUS[149,3]
RBRACK[149,4]
LBRACE[149,5]
DOT[149,6]
ID(extendstkU13)[149,7]
TYPE_VOID[150,1]
MINUS[150,5]
SEMICOLON[153,3]
LT[153,4]
INT(4)[153,5]
RBRACK[154,1]
COMMA[154,2]
INT(21)[155,1]
LBRACE[156,2]
GT[156,3]
LBRACE[156,4]
ID(array8)[156,5]
LBRACE[157,1]
LT[157,3]
LPAREN[157,4]
STRING("s")[157,5]
ID(y2classclass)[157,8]
RBRACK[157,20]
TYPE_INT[157,21]
LBRACK[157,24]
INT(2370)[157,25]
ID(nilstring)[158,1]
RBRACK[158,10]
RBRACE[158,11]
LT[158,12]
MINUS[158,13]
NIL[158,14]
PLUS[158,17]
INT(46)[158,18]
MINUS[159,1]
LT[162,3]
PLUS[162,4]
LBRACK[162,5]
LT[162,6]
MINUS[162,7]
LT[162,8]
INT(4)[162,9]
SEMICOLON[163,1]
RBRACK[163,2]
SEMICOLON[163,3]
RBRACE[163,4]
ID(stringnil21)[163,5]
RBRACK[164,1]
DOT[164,2]
RBRACE[164,3]
TYPE_VOID[165,1]
DOT[165,5]
RPAREN[165,6]
ID(QTstring)[165,7]
EQ[165,15]
RBRACK[165,17]
INT(3645)[165,18]
INT(980)[166,1]
SEMICOLON[167,1]
COMMA[167,2]
NIL[167,3]
COMMA[167,6]
EQ[167,7]
INT(7008)[167,8]
INT(537)[168,1]
RBRACE[169,1]
INT(737)[169,2]
INT(7302)[170,1]
MINUS[171,1]
LT[171,2]
ID(nilint)[171,3]
GT[171,9]
TYPE_STRING[171,10]
RPAREN[171,16]
STRING("JDU")[171,17]
LBRACK[171,22]
LPAREN[171,23]
LBRACK[172,1]
MINUS[172,2]
LBRACK[172,3]
DOT[172,4]
EQ[172,5]
TYPE_STRING[172,6]
PLUS[172,13]
TYPE_STRING[172,14]
LBRACE[172,20]
LBRACK[173,1]
INT(54)[173,2]
GT[174,1]
INT(6870)[174,2]
DOT[175,1]
PLUS[175,2]
LPAREN[175,3]
ARRAY[175,4]
MINUS[175,9]
PLUS[175,10]
MINUS[175,11]
RBRACK[175,12]
ASSIGN[175,13]
RPAREN[175,15]
MINUS[175,16]
ID(classextends)[175,17]
PLUS[175,29]
ASSIGN[175,30]
STRING("yV")[175,32]
INT(888)[175,36]
RBRACK[177,1]
INT(67)[177,2]
INT(5513)[178,2]
TYPE_VOID[179,1]
EQ[179,5]
EQ[179,6]
DOT[179,7]
NIL[179,8]
RPAREN[180,1]
INT(6)[180,2]
ID(NNc)[183,1]
RPAREN[183,4]
ID(void389)[183,5]
RBRACK[184,1]
GT[184,2]
INT(9083)[184,3]
INT(472)[185,1]
LBRACE[187,1]
DOT[187,2]
ARRAY[188,1]
LBRACK[188,6]
EXTENDS[188,7]
LT[188,14]
TYPE_INT[189,11]
RBRACE[189,15]
RBRACE[189,16]
RBRACK[189,17]
NIL[189,18]
ASSIGN[189,21]
DOT[189,23]
ASSIGN[189,24]
RPAREN[190,1]
ID(nil0)[190,2]
RPAREN[192,1]
GT[192,2]
MINUS[192,3]
INT(9121)[192,4]
EQ[193,1]
ID(rSPcintclass)[193,2]
SEMICOLON[193,14]
LBRACE[194,1]
EXTENDS[195,1]
EQ[195,8]
TYPE_VOID[195,9]
RPAREN[195,13]
ASSIGN[195,14]
ID(nil9385)[195,16]
DOT[196,1]
SEMICOLON[196,2]
EXTENDS[196,3]
ID(OB)[201,5]
LBRACK[201,7]
LPAREN[201,8]
LT[201,9]
RPAREN[201,10]
INT(85)[201,11]
TYPE_STRING[202,1]
RBRACK[202,7]
TYPE_INT[202,8]
PLUS[202,11]
DOT[202,13]
LBRACE[202,14]
RBRACK[202,15]
INT(7)[202,16]
ID(arrayextendsclass45)[203,1]
DOT[204,1]
STRING("")[204,2]
LT[204,4]
GT[204,5]
INT(163)[204,6]
GT[205,1]
TYPE_INT[205,2]
EQ[205,5]
ID(stringclass)[205,6]
LBRACK[205,17]
ID(stringint)[211,10]
LBRACE[211,19]
STRING("pEmeZPX")[211,20]
LT[211,29]
RPAREN[211,30]
TYPE_INT[211,31]
RBRACK[211,34]
ID(classextends)[211,35]
PLUS[211,47]
TYPE_STRING[211,48]
LPAREN[211,54]
INT(7828)[211,55]
MINUS[212,1]
TYPE_VOID[212,2]
LT[212,6]
PLUS[213,1]
EXTENDS[213,2]
LT[213,9]
ASSIGN[213,10]
TYPE_INT[213,12]
RBRACE[213,15]
INT(981)[213,16]
ASSIGN[214,1]
SEMICOLON[214,3]
ID(g4B)[214,4]
MINUS[215,1]
COMMA[215,2]
LT[215,3]
LT[215,4]
EQ[215,5]
ID(nilnilextends)[216,7]
MINUS[217,2]
TYPE_INT[217,3]
RBRACK[217,6]
INT(47)[217,7]
RBRACE[219,1]
LBRACK[219,2]
ID(string940)[224,4]
LPAREN[225,1]
ARRAY[225,2]
STRING("")[227,4]
INT(2834)[227,6]
CLASS[228,1]
RPAREN[228,6]
INT(620)[228,7]
RBRACE[229,1]
INT(898)[230,1]
PLUS[231,1]
RBRACE[231,2]
RPAREN[231,3]
LT[231,4]
RBRACE[231,5]
PLUS[239,3]
SEMICOLON[239,4]
ID(extends614)[239,5]
STRING("TtK")[240,1]
LPAREN[240,6]
LPAREN[240,7]
ID(nilclass)[240,8]
SEMICOLON[240,16]
LBRACE[240,17]
TYPE_STRING[240,18]
DOT[240,24]
PLUS[240,25]
DOT[240,26]
SEMICOLON[240,27]
INT(6)[240,28]
CLASS[241,1]
SEMICOLON[241,6]
MINUS[241,7]
TYPE_STRING[241,8]
STRING("U")[241,14]
DOT[241,17]
ID(S389VCwint)[241,18]
INT(37)[242,1]
LPAREN[243,1]
MINUS[243,9]
SEMICOLON[243,10]
ID(extendsvoid)[243,11]
MINUS[244,1]
TYPE_VOID[244,2]
LT[244,6]
EXTENDS[244,7]
LBRACK[246,13]
LBRACK[246,14]
DOT[246,15]
PLUS[246,16]
ARRAY[246,17]
LBRACE[246,22]
ID(k)[246,23]
RBRACE[246,24]
LBRACE[247,6]
MINUS[247,7]
TYPE_STRING[250,3]
STRING("uu")[250,9]
MINUS[250,13]
INT(4)[250,14]
COMMA[251,1]
INT(735)[251,2]
INT(853)[252,1]
ARRAY[253,1]
CLASS[254,1]
LPAREN[254,6]
SEMICOLON[254,7]
ID(stringclass)[254,8]
RBRACK[254,19]
PLUS[254,20]
ASSIGN[255,1]
RPAREN[255,3]
ID(voidclass)[255,4]
DOT[255,13]
RPAREN[255,14]
NIL[255,15]
MINUS[255,18]
MINUS[255,19]
STRING("urq")[255,20]
RBRACE[256,1]
LBRACE[256,2]
RBRACK[256,3]
STRING("")[256,4]
MINUS[256,6]
ID(Tostring)[256,7]
EQ[256,15]
ID(intarray)[257,1]
COMMA[257,9]
INT(5)[257,10]
ID(arraynil)[258,1]
ASSIGN[258,9]
ARRAY[258,11]
COMMA[258,16]
GT[258,17]
LBRACE[258,18]
LBRACE[258,19]
COMMA[259,5]
EQ[259,6]
CLASS[259,7]
SEMICOLON[259,12]
GT[259,24]
SEMICOLON[259,25]
INT(4992)[259,26]
DOT[260,1]
INT(1)[260,2]
LBRACK[261,1]
ID(extendsvFsHBGMtK)[263,1]
RBRACK[263,17]
GT[263,18]
STRING("Wz")[263,19]
INT(3)[263,23]
LBRACK[264,1]
ID(arrayarray)[264,2]
COMMA[264,12]
MINUS[265,1]
ID(nilvoid)[265,2]
LT[265,9]
LBRACK[265,10]
ID(t0)[265,12]
INT(3)[266,1]
LT[267,1]
RBRACE[268,1]
INT(7818)[268,2]
LBRACK[269,2]
RBRACE[269,3]
RBRACE[269,4]
RBRACK[269,5]
ID(nilnil)[269,6]
LPAREN[269,12]
RBRACE[269,13]
ID(extendsextends)[269,14]
LT[269,28]
RBRACE[269,29]
CLASS[269,30]
LPAREN[269,35]
LBRACE[269,36]
INT(47)[269,37]
EXTENDS[270,1]
RBRACE[270,8]
LT[271,1]
EQ[271,2]
EQ[271,3]
INT(5679)[271,4]
PLUS[272,1]
MINUS[272,2]
LT[272,3]
LBRACK[272,4]
ID(class80)[272,6]
ID(Mtqj4KkBUV)[273,1]